code/modules/surgery/organs/internal/eyes/_eyes.dm

```diff
@@ -461,6 +461,8 @@
 /// Animates one eyelid at a time, thanks BYOND and thanks animation chains
 /obj/item/organ/eyes/proc/animate_eyelid(obj/effect/abstract/eyelid_effect/eyelid, mob/living/carbon/human/parent, sync_blinking = TRUE, list/anim_times = null)
 	. = list()
+	if(!CONFIG_GET(flag/blinking)) return // BUBBER EDIT - CONFIG BLINKING
```

Comment:
might be better to put this below in `animate_eyelids`, will just mean it gets interrupted a little earlier in the call stack

Reply:
I just realized that this also breaks rapid eyeblinks ugh